api/swagger.json

```
@@ -0,0 +1,1304 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "description": "This is the API specification for Chapter",
+    "version": "1.0.0",
+    "title": "Chapter",
+    "termsOfService": "http://chapter.io/terms",
+    "contact": {
+      "email": "author@chapter.io"
```

Let's use a more realistic domain for now
```suggestion
      "email": "chapter+dev@freecodecamp.org"
```